To: vim_dev@googlegroups.com
Subject: Patch 7.3.199
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.199
Problem:    MS-Windows: Compilation problem of OLE with MingW compiler.
Solution:   Put #ifdef around declarations. (Guopeng Wen)
Files:	    src/if_ole.h


*** ../mercurial/vim73/src/if_ole.h	2010-05-15 21:22:11.000000000 +0200
--- src/if_ole.h	2011-05-25 12:05:50.000000000 +0200
***************
*** 46,53 ****
  /* header files for imported files */
  #include "oaidl.h"
  
! void __RPC_FAR * __RPC_USER MIDL_user_allocate(size_t);
! void __RPC_USER MIDL_user_free( void __RPC_FAR * );
  
  #ifndef __IVim_INTERFACE_DEFINED__
  #define __IVim_INTERFACE_DEFINED__
--- 46,56 ----
  /* header files for imported files */
  #include "oaidl.h"
  
! #ifndef __MIDL_user_allocate_free_DEFINED__
! #define __MIDL_user_allocate_free_DEFINED__
!     void __RPC_FAR * __RPC_USER MIDL_user_allocate(size_t);
!     void __RPC_USER MIDL_user_free( void __RPC_FAR * );
! #endif
  
  #ifndef __IVim_INTERFACE_DEFINED__
  #define __IVim_INTERFACE_DEFINED__
*** ../vim-7.3.198/src/version.c	2011-05-19 18:26:34.000000000 +0200
--- src/version.c	2011-05-25 12:08:54.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     199,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
104. When people ask about the Presidential Election you ask "Which country?"

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
